src/parsing/astHelper.ts

```diff
@@ -5,10 +5,12 @@ import {
 	ValueType,
 } from '../expressions/dataTypes/Value';
 import { SourceRange } from '../expressions/debug/StackTraceGenerator';
+import { BinaryEvaluationFunction } from '../typeInference/binaryEvaluationFunction';
 
 type QName = { localName: string; namespaceURI: string | null; prefix: string };
 
-export interface IAST extends Array<string | object | SourceRange | IAST> {
+export interface IAST
+	extends Array<string | object | SourceRange | IAST | SequenceType | BinaryEvaluationFunction> {
```

Comment:
That last one is no longer there, right?

Reply:
This has been fixed in our current development branch